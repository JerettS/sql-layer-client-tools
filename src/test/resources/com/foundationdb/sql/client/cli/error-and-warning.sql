SET foo TO 'bar';
SELECT foo FROM bar;
SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED;
