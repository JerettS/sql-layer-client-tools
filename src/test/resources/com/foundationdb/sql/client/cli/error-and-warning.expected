ERROR: Unrecognized configuration parameter `foo`

ERROR: Cannot find the table `test`.`bar`

WARNING: ISOLATION LEVEL READ UNCOMMITTED ignored; remains SERIALIZABLE
ROWS: 0

